packages/backend/test/peripherals/database/ForcedTradeOfferRepository.test.ts

```diff
@@ -96,16 +96,94 @@ describe(ForcedTradeOfferRepository.name, () => {
     expect(actual).toEqual(submitted)
   })
 
-  it('get initial count', async () => {
-    await repository.add(fakeInitialOffer())
-    expect(await repository.initialCount()).toEqual(1)
-
-    const initial = fakeInitialOffer()
-    const id = await repository.add(initial)
-    expect(await repository.initialCount()).toEqual(2)
-
-    await repository.save({ ...initial, id, accepted: fakeAccepted() })
-    expect(await repository.initialCount()).toEqual(1)
+  it('returns initial offers', async () => {
```

Comment:
The test is good, but splitting it into multiple would allow granularity in the output if something goes wrong